### ISSUE TITLE
Bring up to speed with `datalad-extension-template`

### ISSUE THREAD

**Issue by jsheunis** [datalad/datalad-neuroimaging#99]
See: https://github.com/datalad/datalad-extension-template

Since this extension was created before more recent updates to the extension template, some aspects of the repo could be outdated.

**Comment by jsheunis** [comment]
Unless I'm missing some context from on the history of this extension, I'd say we should keep it as close to the extension template as possible.


> addition of two github workflows (does the docs workflow mean that the docs build process changes completely?)

I would add both, since they are part of the standard template. Let's see what the response is to the docs build process #98.   If it requires a different setup to what is currently implemented here, that change would be easy.

> addition of codeclimate (is external setup needed? is it wanted here?)

I don't think it would hurt to have the codeclimate config here. I'm guessing it needs some external setup/linking for it to start working, so that step can always be done at a later stage if it's not needed now.

> I'm missing packaging-related knowledge, may try but feels risky to touch

Yeah I'm also hesitant here. I'd try and duplicate the extension-template, but I don't know what (required) functionality I'd be dropping unintentionally by removing existing parts. I'd get extra input here. @datalad/developers can anyone help clarify which current code/files in `datalad-neuroimaging` are necessary to keep in addition to updating content to mirror the extension setup?

**Comment by mih** [comment]
I am fully in line for @jsheunis. Few additional pointers:

Code climate report: https://codeclimate.com/github/datalad/datalad-neuroimaging (no additional setup required, but could be tuned)

Re unintentional removing: I wouldn't worry about it much. First, it is useful to question everything that the old setup installed -- who knows if it is still relevant. Once the update PR is done. It is easy to compare the outcomes of `python setup.py (s|b)dist` pre and post update to discover and discuss and potential issues.
